tools/dts/maaxboard.dts

```diff
@@ -0,0 +1,1849 @@
+				interrupt-controller;
+				broken-wake-request-signals;
+				#interrupt-cells = <0x03>;
+				phandle = <0x01>;
+
+				pgc {
+					#address-cells = <0x01>;
+					#size-cells = <0x00>;
+
+					power-domain@0 {
+						#power-domain-cells = <0x00>;
+						reg = <0x00>;
+						phandle = <0x21>;
+					};
+
+					power-domain@1 {
+						#power-domain-cells = <0x00>;
+						reg = <0x01>;
+						power-domains = <0x1b>;
+						phandle = <0x43>;
+					};
+
+					power-domain@2 {
+						#power-domain-cells = <0x00>;
+						reg = <0x02>;
+					};
+
+					power-domain@3 {
+						#power-domain-cells = <0x00>;
+						reg = <0x03>;
+						phandle = <0x41>;
+					};
+
+					power-domain@4 {
+						#power-domain-cells = <0x00>;
+						reg = <0x04>;
+					};
+
+					power-domain@5 {
+						#power-domain-cells = <0x00>;
+						reg = <0x05>;
+						clocks = <0x02 0xd7 0x02 0x66 0x02 0x6f 0x02 0x70>;
+						power-supply = <0x1c>;
+						phandle = <0x3e>;
+					};
+
+					power-domain@6 {
+						#power-domain-cells = <0x00>;
+						reg = <0x06>;
+						power-supply = <0x1d>;
+						phandle = <0x44>;
+					};
+
+					power-domain@7 {
+						#power-domain-cells = <0x00>;
+						reg = <0x07>;
+					};
+
+					power-domain@8 {
+						#power-domain-cells = <0x00>;
+						reg = <0x08>;
+						phandle = <0x2e>;
+					};
+
+					power-domain@9 {
+						#power-domain-cells = <0x00>;
+						reg = <0x09>;
+						phandle = <0x39>;
+					};
+
+					power-domain@a {
+						#power-domain-cells = <0x00>;
+						reg = <0x0a>;
+						phandle = <0x1b>;
+					};
+				};
+			};
+		};
+
+		bus@30400000 {
+			compatible = "fsl,imx8mq-aips-bus\0simple-bus";
+			#address-cells = <0x01>;
+			#size-cells = <0x01>;
+			ranges = <0x30400000 0x30400000 0x400000>;
+
+			pwm@30660000 {
+				compatible = "fsl,imx8mq-pwm\0fsl,imx27-pwm";
+				reg = <0x30660000 0x10000>;
+				interrupts = <0x00 0x51 0x04>;
+				clocks = <0x02 0xbf 0x02 0xbf>;
+				clock-names = "ipg\0per";
+				#pwm-cells = <0x02>;
+				status = "disabled";
+				pinctrl-names = "default";
+				pinctrl-0 = <0x1e>;
+				phandle = <0x49>;
+			};
+
+			pwm@30670000 {
+				compatible = "fsl,imx8mq-pwm\0fsl,imx27-pwm";
+				reg = <0x30670000 0x10000>;
+				interrupts = <0x00 0x52 0x04>;
+				clocks = <0x02 0xc0 0x02 0xc0>;
+				clock-names = "ipg\0per";
+				#pwm-cells = <0x02>;
+				status = "disabled";
+			};
+
+			pwm@30680000 {
+				compatible = "fsl,imx8mq-pwm\0fsl,imx27-pwm";
+				reg = <0x30680000 0x10000>;
+				interrupts = <0x00 0x53 0x04>;
+				clocks = <0x02 0xc1 0x02 0xc1>;
+				clock-names = "ipg\0per";
+				#pwm-cells = <0x02>;
+				status = "disabled";
+			};
+
+			pwm@30690000 {
+				compatible = "fsl,imx8mq-pwm\0fsl,imx27-pwm";
+				reg = <0x30690000 0x10000>;
+				interrupts = <0x00 0x54 0x04>;
+				clocks = <0x02 0xc2 0x02 0xc2>;
+				clock-names = "ipg\0per";
+				#pwm-cells = <0x02>;
+				status = "disabled";
+			};
+
+			timer@306a0000 {
+				compatible = "nxp,sysctr-timer";
+				reg = <0x306a0000 0x20000>;
```

Comment:
0x20000 is incorrect. The memory mapped area for this timer is larger than 0x20000: it should be 0x30000 (rounded up).
Fix in overlay-maaxboard.dts.

Reply:
Corrected in the [overlay file](https://github.com/seL4/seL4/pull/829/commits/f0da338534e194fee4865620d85e60f490d11129).